skyportal/handlers/api/photometry.py

```diff
@@ -1116,16 +1116,12 @@ def patch(self, photometry_id):
                 )
             ).first()
             if phot_stat is not None:
-                all_phot = (
-                    session.scalars(
-                        Photometry.select(session.user_or_token)
-                        .where(Photometry.obj_id == photometry.obj_id)
-                        .distinct()
-                    )
-                    .unique()
-                    .all()
-                )
+                all_phot = session.scalars(
+                    sa.select(Photometry).where(Photometry.obj_id == photometry.obj_id)
+                ).all()
                 phot_stat.full_update(all_phot)
+                for phot in all_phot:
+                    session.expunge(phot)
```

Comment:
I guess this was the addition?

Reply:
Among other things. You can see what changed since your last review by looking at the commits picker in review mode.